_posts/convars/2020-10-25-mom_gamemode_override.md

```diff
@@ -0,0 +1,28 @@
+---
+title: mom_gamemode_override
+category: var
+tags:
+  - gamemode
+minimum_value: 0
+maximum_value: Same as gamemode count
+default_value: 0
+---
+
+Forces gamemode to the specified gamemode number given; must not be in-game to change.  
+If this cvar isn't set to its default, your times will not submitted upon map completion.
+
+**Note:** If you wish to use this cvar with official maps, you should instead use the "Start Map In Specific Gamemode" option in the Map Selector's context menu.
```

Comment:
Can use notices for a better effect. Note this won't work unless the branch is rebased as there was recent changes to them.
```suggestion
{:.notice--info}
If you wish to use this cvar with official maps, you should instead use the "Start Map In Specific Gamemode" option in the Map Selector's context menu.
```